### PR TITLE
Bump airflow version to 1.10.4

### PR DESCRIPTION
**Pull request by andriisoldatenko**
**TODO:**
- [x] Update airflow Dockerfile
- [x] Need sync apache/airflow with astronomer/airflow fork
cc @schnie @andrewhharmon 

https://github.com/astronomer/airflow/commits/v1-10-4

### CODE REVIEW

File: docker/airflow/1.10.4/Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ LABEL io.astronomer.docker.component="airflow"
 LABEL io.astronomer.docker.airflow.version="1.10.3"
```
Comment (andrewhharmon) [inline docker/airflow/1.10.4/Dockerfile]: should this bump to 1.10.4?

Comment (schnie) [reply docker/airflow/1.10.4/Dockerfile]: Yep

Comment (andriisoldatenko) [reply docker/airflow/1.10.4/Dockerfile]: fixed.

**Comment by andriisoldatenko** [comment]
@schnie PTAL
btw what's missing from my side?

**Comment by schnie** [comment]
Looks good. Ready to merge once we have that branch released.

**Comment by andriisoldatenko** [comment]
image

**Comment by andriisoldatenko** [comment]
cc @ashb

**Comment by schnie** [comment]
@andriisoldatenko can you bump airflow version in `Makefile`?

**Comment by andriisoldatenko** [comment]
@schnie ptal

**Pull request merged**